src/components/AddRepsModal/LiftInfo/Links.tsx

```diff
@@ -35,11 +35,11 @@ const Links = (props: Prop) => {
         className="pt-1 mx--1"
         height={linksContainerHeight}
       >
-        {links.map((link, ind) => (
+        {links.map((link, index) => (
           <div
             className="d-flex align-items-start px-1"
             style={{ height: LINK_ELEMENT_HEIGHT }}
-            key={ind}
+            key={`${link.text}_${link.url}`}
```

Comment:
I am still not sure why this was a problem... and why this isn't a problem with `CustomSetsReps`. Doesn't the list get rerendered on every delete, which would reindex the elements? Or is `React` too smart and only rerenders the changes (which in this case means removing one element from the DOM and not touching the rest)?

Reply:
Ok, posting this here for the sake of history:
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318